core/capabilities/ccip/common/addresscodec.go

```diff
@@ -47,3 +50,17 @@ func (ac AddressCodec) AddressStringToBytes(addr string, chainSelector cciptypes
 
 	return codec.AddressStringToBytes(addr)
 }
+
+// OracleIDAsAddressBytes returns valid random address bytes for a given chain selector.
+func (ac AddressCodec) OracleIDAsAddressBytes(oracleID uint8, chainSelector cciptypes.ChainSelector) ([]byte, error) {
```

Comment:
Should we note that this is to support a temporary hack? Or would it have other use cases?

Reply:
This is the only use case I can think of, but not sure how temporary it'd be. There is logic deep inside libocr surrounding the changes I made in this PR and I don't really see it going away anytime soon.

Reply:
I can definitely add a comment about it though.